docs/src/complex.md

```diff
@@ -56,3 +56,40 @@ julia> wirtinger(x -> 3x^2 + 2x + 1, 1+2im)
 julia> wirtinger(x -> abs2(x), 1+2im)
 (1.0 - 2.0im, 1.0 + 2.0im)
 ```
+
+The gradient definition Zygote uses can also be expressed in terms of the [Wirtinger calculus](https://en.wikipedia.org/wiki/Wirtinger_derivatives) using the operators ``\frac{\partial}{\partial z}`` and ``\frac{\partial}{\partial z^*}``:
+
+```math
+f: \mathbb{C} \rightarrow \mathbb{R}, \qquad
```

Comment:
Don't you mean C->C here? The below references `Re(f)` and `f*`.

Reply:
It is C->R, but since `f` is real, I'm using `f = Re(f) = f*` here, to express this in terms of the Wirtinger derivatives